frontend/src/components/Classroom/ClassroomList.vue

```diff
@@ -10,18 +13,51 @@ const props = defineProps({
     default: () => [],
   },
 });
-const classroomFields = {
-  name: 'Name',
-  capacity: 'Capacity',
-  createdAt: 'Created At',
-};
+const classroomFields = [
+  {
+    displayName: 'Name',
+    property: 'name',
+    isEditable: true,
+  },
+  {
+    displayName: 'Capacity',
+    property: 'capacity',
+    isEditable: true,
+  },
+  {
+    displayName: 'Created At',
+    property: 'createdAt',
+    isEditable: false,
+  },
+];
+const classroomEditingId = ref(null);
 
 const emit = defineEmits(['reload']);
 const filteredClassrooms = computed(() => {
   const searchQueryLowercased = searchQuery.value.toLowerCase();
   const filtered = props.classrooms.filter(classroom => classroom.name.toLowerCase().includes(searchQueryLowercased));
   return filtered;
 });
+const isToEdit = classroomId => {
```

Comment:
Nitpick: this name seems to me to be a little convoluted and difficult to understand. Should it be something like 'isBeingEdited' or something like that?